charts/prometheus-adapter/templates/deployment.yaml

```diff
@@ -42,7 +42,11 @@ spec:
       {{- end}}
       containers:
       - name: {{ .Chart.Name }}
+        {{- if .Values.image.sha256 }}
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}@sha256:{{ .Values.image.sha256 }}"
```

Comment:
Why can't you just populate the sha256 on the existing `image.tag`? The sha256 is specific to any version so you would have to adjust it for every chart release anyway. You don't gain anything by making it separate. In general I'm against increased complexity if there's a valid way to accomplish the same thing.

```yaml
image:
  tag: v0.9.1@sha256:abcd...
```

Reply:
You're right. I'm ok with that strategy :ok_hand: 
Thanks for the quick response